src/main/kotlin/it/pagopa/wallet/scheduler/exceptions/LockException.kt

```diff
@@ -0,0 +1,4 @@
+package it.pagopa.wallet.scheduler.exceptions
+
+open class LockException(message: String, throwable: Throwable? = null) :
```

Comment:
```suggestion
sealed class LockException(message: String, throwable: Throwable? = null) :
```

Make this class sealed so that all implementor are known at compile time